diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index a6e7c9b..e890fca 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -5,6 +5,7 @@ static_library("browser") {
     "child_process_host_flags.h",
     "helium/deterministic/deterministic_session_service.cc",
     "helium/deterministic/deterministic_session_service.h",
+    "helium/deterministic/step_transaction_types.h",
     "helium/deterministic/virtual_time_controller.cc",
     "helium/deterministic/virtual_time_controller.h",
     "chrome_browser_field_trials.cc",
diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 689b40e..6caa201 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -2,6 +2,7 @@ test("unit_tests") {
   sources = [
     "../browser/history_clusters/history_clusters_tab_helper_unittest.cc",
     "../browser/helium/deterministic/deterministic_session_service_unittest.cc",
+    "../browser/helium/deterministic/step_transaction_types_unittest.cc",
     "../browser/helium/deterministic/virtual_time_controller_unittest.cc",
     "../browser/idle/idle_detection_permission_context_unittest.cc",
   ]
